src/main/java/com/minecolonies/client/render/RenderBipedCitizen.java

```diff
@@ -44,6 +48,7 @@
     public RenderBipedCitizen(RenderManager renderManagerIn)
     {
         super(renderManagerIn, defaultModelMale, 0.5F);
+        this.addLayer(new LayerBipedArmor(this));
```

Comment:
We could move wearing armor as a whole down some ai layers if that is possible. Maybe (later) add an equipment gui for workers